src/constructs/lambda/lambda.test.ts

```diff
@@ -172,4 +172,24 @@ describe("The GuLambdaFunction class", () => {
       Timeout: 30,
     });
   });
+
+  it("should support bucket override", () => {
+    const stack = simpleGuStackForTesting();
+
+    new GuLambdaFunction(stack, "lambda", {
+      fileName: "my-app.jar",
+      bucketNamePath: "/example-parameter-path", // the override
+      handler: "handler.ts",
```

Comment:
`handler.ts` in a Java 8 runtime...

![img](https://media.giphy.com/media/3ohhwF34cGDoFFhRfy/giphy.gif)

Reply:
😬 - haha hadn't noticed this. In my defence, I copied this from the tests above it, all of which do the same thing!